templater/jobs/postsubmits.go

```diff
@@ -0,0 +1,483 @@
+package jobs
+
+func getPostsubmitsList() map[string]map[string]JobConfig {
+	return map[string]map[string]JobConfig{
+		"aws/eks-distro":               getEksDistroPostsubmitList(),
+		"aws/eks-distro-build-tooling": getEksDistroBuildToolingPostsubmitList(),
+	}
+}
+
+func getEksDistroPostsubmitList() map[string]JobConfig {
```

Comment:
A big change I know, but I wonder if it would be nicer if we put these in yaml files and then just read all the yamls into the go structs?  Probably would be easier to update for people?

Also, for all the generated yaml files lets add a big note at the top about how these are auto generated and not to modify them, instead linking to the doc on how to run the script to update

Reply:
Done